react/finger-exercises/redux/02 - react-redux/src/app/screens/App/index.js

```diff
@@ -75,4 +98,8 @@ class App extends Component {
   }
 }
 
+App.propTypes = {
+  data: PropTypes.arrayOf(listBooksPropType)
```

Comment:
If you import directly arrayOf propType here you can change this for something like.

```
 data: arrayOf(listBooksPropType)
```